vizro-core/docs/pages/user_guides/layouts.md

```diff
@@ -7,7 +7,7 @@ The [`Page`][vizro.models.Page] model accepts the `layout` argument, where you c
 
 ## Using the default layout
 The `layout` argument of the [`Page`][vizro.models.Page] model is optional. If no layout is specified, all charts/components
-will automatically be stacked underneath each other. If that is your desired layout, you can create your charts/components without providing a [`Layout`][vizro.models.Layout].
+will automatically be stacked horizontally. If that is your desired layout, you can create your charts/components without providing a [`Layout`][vizro.models.Layout].
```

Comment:
```suggestion
will automatically be stacked in rows down the page. If that is your desired layout, you can create your charts/components without providing a [`Layout`][vizro.models.Layout].
```

Due to the slight ambiguity mentioned below, best to start off with non-technical language when this is first mentioned I think.